Assignment/DenyTeacher.txt

```diff
@@ -0,0 +1,12 @@
+trigger DenyTeacher on Teacher__c (before insert, before update)
+{
+    for(Teacher__c t: Trigger.new)
+    {
+        if(t.subjects__c.contains('Hindi'))
+        {
+            t.addError(t.Name + '\tTeaher cannot teach hindi');
```

Comment:
Error message not correct, it should be - Teacher teaching Hindi is not allowed to insert/update.

Reply:
Fixed.